.gitlab-ci.yml

```diff
@@ -203,13 +245,13 @@ publish-docs:
     - linux-docker
 
 build-android:
-  stage:                           optional
+  stage:                           build
   image:                           parity/rust-android:gitlab-ci
   variables:
     CARGO_TARGET:                  armv7-linux-androideabi
   dependencies:                    []
   script:
-    - scripts/gitlab/build-unix.sh
+    - scripts/gitlab/build-linux.sh
   tags:
     - linux-docker
   allow_failure:                   true
```

Comment:
who will resolve the issues when they are already merged?

Reply:
fixed